Demo/Demo/DemoSettings/DemoType.swift

```diff
@@ -3,24 +3,24 @@ import SwiftUI
 
 enum DemoType: String {
     case card
-    case paypal
+    case payPalWebCheckout
```

Comment:
There's some inconsistency with this naming convention. I'm wondering is `paypal` or `payPal` a better prefix when at the beginning of a camel case variable?

Reply:
I think we want to do `payPal` - I feel like we talked about this with Sammy :pour-one-out: at some point and all agreed on that.

Reply:
Nice aight. I just fixed camel casing throughout to keep it consistent.